frontend/selectors/index.js

```diff
@@ -11,3 +11,12 @@ export const isBurgerIconVisible = createSelector(
   currentRoute => !burgerIconBlacklist.find(entry => currentRoute.pattern.startsWith(entry))
 );
 
+export const isHome = createSelector(
+  getCurrentRoute,
+  (route) => {
+    if (route.pattern === '/') {
```

Comment:
please use the PWA constant (INDEX_PATH) here

Reply:
Done.